src/Workspaces/Core/Portable/Microsoft.CodeAnalysis.Workspaces.csproj

```diff
@@ -29,6 +29,7 @@
   </ItemGroup>
   <ItemGroup Label="Linked Files">
     <Compile Include="..\..\..\CodeStyle\Core\CodeFixes\FixAllContextHelper.cs" Link="CodeFixes\FixAllOccurrences\FixAllContextHelper.cs" />
+    <Compile Include="..\..\..\Compilers\Server\VBCSCompiler\MemoryHelper.cs" Link="Utilities\CompilerUtilities\MemoryHelper.cs" />
```

Comment:
Why are we linking this into the Workspaces layer? The file is using P/Invokes and therefore is very non-portable. I'd be very worried somebody calling this on a non-Windows platform.

Reply:
I'll move it up to the VS layer